### PR TITLE
Add milk and cookies option to jacket

### PR DESCRIPTION
**Pull request by danessh**

### CODE REVIEW

File: english/jacket/marshmallow-jacket.md [hunk 1]
```diff
@@ -3,3 +3,4 @@ creating a new jacket out of marshmallows. Your hope is to make this one less re
 You get started right away melting and cooling marshmallows in to the form of a jacket.
 
 Do you:
+Give up and have [milk and cookies](../milk-and-cookies/jacket-cookie.md)
```
Comment (casey-collab) [inline english/jacket/marshmallow-jacket.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/jacket/milk-and-cookies/jacket-cookie.md [hunk 2]
```diff
@@ -0,0 +1 @@
+As you take the last bite of a 32 pack of Oreo's you realize the milk was actually perfectly liquefied marshmallows.
```
Comment (casey-collab) [inline english/jacket/milk-and-cookies/jacket-cookie.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by danessh** [comment]
Please review this, @casey-collab

File: english/jacket/marshmallow-jacket.md [hunk 3]
```diff
@@ -3,3 +3,4 @@ creating a new jacket out of marshmallows. Your hope is to make this one less re
 You get started right away melting and cooling marshmallows in to the form of a jacket.
 
 Do you:
+Give up and have [milk and cookies](..jacket/milk-and-cookies/jacket-cookie.md)
```
Comment (casey-collab) [inline english/jacket/marshmallow-jacket.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/jacket/milk-and-cookies/jacket-cookie.md (hunk 2 above)
Comment (casey-collab) [inline english/jacket/milk-and-cookies/jacket-cookie.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by danessh** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :four_leaf_clover:

**Pull request merged**